tensorflow/core/util/rocm_sparse.cc

```diff
@@ -207,6 +212,8 @@ Status GpuSparse::Csr2coo(const int* csrRowPtr, int nnz, int m,
   return Status::OK();
 }
 
+#if TF_ROCM_VERSION < 40200
```

Comment:
All of the code within the if block will stop working once we switch to ROCm 4.2...is that the intent here?

Reply:
Yes, that is the intent. Makes it more in line with the CUDA implementation. https://github.com/ROCmSoftwarePlatform/tensorflow-upstream/blob/develop-upstream/tensorflow/core/util/cuda_sparse.cc#L389-L431